src/main/java/sergey/MyFilter.java

```diff
@@ -0,0 +1,25 @@
+package sergey;
+
+import javax.servlet.*;
+import javax.servlet.annotation.WebFilter;
+import java.io.IOException;
+
+@WebFilter(
+        urlPatterns = {"/calc/*"}
+)
+public class MyFilter implements javax.servlet.Filter {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'javax.servlet.Filter' due to existing import 'javax.servlet.*'](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4723796)